pippo-content-type-parent/pippo-csv/pom.xml

```diff
@@ -44,7 +44,7 @@
             <groupId>org.projectlombok</groupId>
             <artifactId>lombok</artifactId>
             <version>1.18.6</version>
-            <scope>test</scope>
+            <scope>provided</scope> <!-- https://projectlombok.org/setup/maven -->
```

Comment:
The scope `provided` was necessary to run the tests and is the recommended scope in the Lombok documentation. See https://projectlombok.org/setup/maven